src/api/rest/v1/providers/controller.ts

```diff
@@ -0,0 +1,102 @@
+import { Request, Response } from 'express'
+import Providers from "../../../../providers"
+import { ProviderHandler } from '../../../../interfaces'
+import CONFIG from '../../../../config'
+
+const log4js = require("log4js")
+const logger = log4js.getLogger()
+
+const SCHEMA_SYNC_LOG = CONFIG.verida.schemas.SYNC_LOG
+
+/**
+ */
+export default class Controller {
+
+    public static async provider(req: Request, res: Response) {
+        const providerId = req.params.providerId
+        const providers = Object.keys(CONFIG.providers)
+
+        const results: any = []
+        for (let p in providers) {
+            const providerName = providers[p]
+            if (providerName != providerId) {
+                continue
+            }
+
+            try {
+                const provider = Providers(providerName)
+                const syncHandlers = await provider.getSyncHandlers()
+                const handlers: ProviderHandler[] = []
+                for (const handler of syncHandlers) {
+                    handlers.push({
+                        id: handler.getName(),
+                        label: handler.getLabel(),
+                        options: handler.getOptions()
+                    })
+                }
+
+                results.push({
+                    name: providerName,
```

Comment:
Should be `id`.
Would be caught by a strong type on `results`, see comment line 19.

Reply:
Fixed, although both are strings so wouldn't have been caught.